src/Compiler/Microsoft.AspNetCore.Razor.Language/test/TestFiles/IntegrationTests/ComponentRuntimeCodeGenerationTest/AtTransitions/TestComponent.codegen.cs

```diff
@@ -27,10 +27,9 @@ protected override void BuildRenderTree(global::Microsoft.AspNetCore.Components.
 #line hidden
 #nullable disable
 
-            __builder.AddContent(0, 
 #nullable restore
-#line (3,6)-(3,7) "x:\dir\subdir\Test\TestComponent.cshtml"
-x
+#line (3,6)-(3,7) 24 "x:\dir\subdir\Test\TestComponent.cshtml"
+__builder.AddContent(0, x
```

Comment:
Do we have some tests where the expression spans multiple lines?

Reply:
We had some in the razor source generator tests incidentally, but I've added an explicit one.